projects/common/src/lib/services/pipeline/pipeline.service.ts

```diff
@@ -1766,27 +1791,24 @@ export class PipelineService {
             });
           }
         } else {
-          let resourceToPush = {
-            'Type': 'file',
-            'Name': resource.name,
-            'Value': this.isBase64(resource.fileContent) ? atob(resource.fileContent) : decodeURIComponent(resource.fileContent),
-            'Encoded': false
-          }
-          switch (resource.type) {
-            case 'processor': {
-              resourceToPush.Name = resourceToPush.Name + (resourceToPush.Name.includes('.py') ? '' : '.py');
-              break
+          if ((resource.type != 'requirement') && (resource.type != 'configuration')) {
```

Comment:
Is there an `else` side to this `if` condition? There might be, but it might be hard to see in the pull request view.

Reply:
There is not. That handling is actually stored elsewhere with these new requirements.